packages/mds-ingest-service/repository/index.ts

```diff
@@ -217,6 +207,7 @@ class IngestReadWriteRepository extends ReadWriteRepository {
 
       if (grouping_type === 'all_events') {
         query.andWhere('events.timestamp >= :start AND events.timestamp <= :end', { start, end })
+        query.limit(limit + 1)
```

Comment:
why `+ 1`?

Reply:
To know whether there’s a next cursor I think.

Reply:
Yup! Exactly.